exp-player/addon/components/exp-player.js

```diff
@@ -148,6 +148,9 @@ able to continue the study.
                 this._transition();
                 this.set('frameIndex', frameIndex + 1);
                 this.set('framePage', 0);
+                this.get('session').set('frameIndex', frameIndex + 1);
+                this.get('session').set('surveyPage', 0);
+                this.get('session').save();
```

Comment:
In particular, remember that this is meant to be a somewhat general infrastructure. Not every experiment on this platform may want to save data as the user goes along- some may want to only save at end. 

We should be careful about modifying the base `exp-player` too much to suit the use case of one particular experiment. (And I say that for my work on Lookit as well... we can learn from past experience here)